### PR TITLE
Format gRPC bidirectional streaming sample and add protobuff tool sample

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: examples/protobuff-tool/helloWorld.proto [hunk 1]
```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+service helloWorld {
+		rpc hello(HelloRequest) returns (HelloResponse);
```
Comment (Shan1024) [inline examples/protobuff-tool/helloWorld.proto]: Seems like these lines are not properly formatted.

**Review by afkham — CHANGES_REQUESTED** [review]
Please make the changes requested by other reivewers.

File: examples/grpc-bidirectional-streaming/client/chat.sample.client.bal [hunk 2]
```diff
@@ -16,7 +17,7 @@ function main(string... args) {
     var res = chatEp->chat(ChatMessageListener);
     match res {
         grpc:error err => {
-            io:print("error");
+            io:print("Unexpected error occured.");
```
[the remark this answers is not in the document]
Comment (Yasara123) [reply examples/grpc-bidirectional-streaming/client/chat.sample.client.bal]: Fixed 86fd958

File: examples/protobuff-tool/helloWorld.proto (hunk 1 above)
Comment (Yasara123) [reply examples/protobuff-tool/helloWorld.proto]: Fixed 86fd958

**Review by afkham — APPROVED** [review]

**Pull request merged**